utils/PUMA_helpers.py

```diff
@@ -5,6 +5,10 @@
 import requests
 from utils.geocode import from_eviction_address
 
+acs_years = ["0812", "1519", "1721"]
+
+year_range = lambda y: f"20{y[:2]}-20{y[2:]}"
+sheet_name = lambda y: f"{y[:2]}-{y[2:]}"
```

Comment:
a more descriptive variable name for `y` would be nice

maybe `years` since the input string would be something like `"1721"`?

then again, I'm not these three things being used in this file or [anywhere else](https://github.com/search?q=repo%3ANYCPlanning%2Fdb-equitable-development-tool+%22PUMA_helpers+import%22&type=code) so maybe to-be-deleted?

Reply:
I think that search is on main? These are being used a fair amount. And sure - I'll rename "y" to "acs_year"

Reply:
Each of those functions are used in about 4 files, acs_years referenced in a dozen or so

Reply:
from github docs: "We currently only support searching for code on the default branch of a repository."

boooooo